draft-ietf-masque-h3-datagram.md

```diff
@@ -28,65 +28,71 @@ author:
+such as unreliable delivery (as offered by {{!DGRAM=I-D.ietf-quic-datagram}})
+to enable desirable features like an unreliable version of the CONNECT method,
+and unreliable delivery in WebSockets {{?RFC6455}} (or its successors).
 
+In {{datagrams}}, this document describes HTTP Datagrams, a convention for
+conveying multiplexed, bidirectional datagrams inside a HTTP connection. They
+are intended for use by HTTP extensions (such as the CONNECT method), and are
+compatible with all versions of HTTP. When the underlying transport protocol
+supports unreliable delivery (such as when the QUIC DATAGRAM extension is
+available in HTTP/3), they use that capability.
 
-This document is structured as follows:
-
-* {{multiplexing}} presents core concepts for multiplexing across HTTP versions.
-* {{format}} defines how QUIC DATAGRAM frames are used with HTTP/3.
-  * {{setting}} defines an HTTP/3 setting that endpoints can use to advertise
-  support of the frame.
-* {{capsule}} introduces the Capsule Protocol and the "data stream" concept.
-  Data streams are initiated using special-purpose HTTP requests, after which
-  Capsules, an end-to-end message, can be sent.
-  * {{datagram-capsule}} defines Datagram Capsule types, along with guidance
-    for specifying new capsule types.
+It also describes the HTTP Capsule Protocol in {{capsule}}, to allow conveyance
+of HTTP Datagrams over other versions of HTTP.
 
 
 ## Conventions and Definitions {#defs}
 
 {::boilerplate bcp14-tagged}
 
 
-# Multiplexing
+# HTTP Datagrams {#datagrams}
 
-All HTTP Datagrams are associated with an HTTP request.
+HTTP Datagrams are a convention for conveying bidirectional and potentially
+unreliable datagrams inside an HTTP connection, with multiplexing when
+possible. All HTTP Datagrams are associated with an HTTP request.
 
-When running over HTTP/3, multiple exchanges of datagrams need the ability to
-coexist on a given QUIC connection. To allow this, the QUIC DATAGRAM frame
-payload starts with an encoded stream identifier that associates the datagram
-with a request stream.
+When HTTP Datagrams are conveyed on a HTTP/3 connection, the QUIC DATAGRAM
+frame is used to achieve these goals, including unreliable delivery; see
+{{format}}. Negotiation is achieved using a SETTING; see {{setting}}.
 
 When running over HTTP/2, demultiplexing is provided by the HTTP/2 framing
-layer. When running over HTTP/1, requests are strictly serialized in the
-connection, therefore demultiplexing is not needed.
+layer, but unreliable delivery is unavailable. HTTP Datagrams are negotiated
+and conveyed using the Capsule Protocol; see {{datagram-capsule}}.
+
+When running over HTTP/1, requests are strictly serialized in the connection,
+and therefore demultiplexing is not available. Unreliable delivery is likewise
+not available. HTTP Datagrams are negotiated and conveyed using the Capsule
+Protocol; see {{datagram-capsule}}.
```

Comment:
```suggestion
When running over HTTP/1, upgrading the connection to convay HTTP datagrams prevents other uses of the connection and therefore (de-)multiplexing not possible. Unreliable delivery is likewise
not available. HTTP Datagrams are negotiated and conveyed using the Capsule
Protocol; see {{datagram-capsule}}.
```
Closes #156

Reply:
This PR is intended to be purely editorial, so let's keep resolution of that separate.